skymap_scanner/client/reco_icetray.py

```diff
@@ -155,12 +153,12 @@ def UncompressGCD(tray, name, base_GCD_path, base_GCD_filename):
             base_filename=base_GCD_filename,
         )
 
-    if GCD_diff_base_handle := get_GCD_diff_base_handle(baseline_GCD_file):
+    if baseline_GCD_handle := get_baseline_GCD_handle(baseline_GCD_file):
         tray.Add(
             UncompressGCD,
             "GCD_uncompress",
             base_GCD_path="",
-            base_GCD_filename=str(GCD_diff_base_handle),
+            base_GCD_filename=str(baseline_GCD_handle),
```

Comment:
Question: is "base" just short for "baseline"?

Reply:
"diff_base" stands for "base GCD from which the GCD diff is calculated". This is equivalent with what we call "baseline GCD" otherwise.